lib/razorpay/payment.rb

```diff
@@ -10,6 +10,46 @@ class Payment < Entity
     def self.request
       Razorpay::Request.new('payments')
     end
+    
+    def self.create_recurring_payment(data={})
+      request.post "create/recurring" , data
+    end
+
+    def self.create_json_payment(data={})
+      request.post "create/json" , data
+    end
+
+    def self.fetch_payment_downtime
+      request.get "downtimes"
+    end
+
+    def self.fetch_payment_downtime_by_id(id)
+      request.get "downtimes/#{id}"
+    end
+
+    def self.fetch_card_details(id)
+      request.get "#{id}/card"  
+    end
+    
+    def fetch_transfer
+      self.class.request.get "#{id}/transfers"  
+    end
+
+    def fetch_refund(refundId)
+      self.class.request.get "#{id}/refunds/#{refundId}"  
+    end
+    
+    def self.fetch_multiple_refund(id, options = {})
+      request.get "#{id}/refunds",options
+    end
+
+    def transfer(options = {})
+      self.class.request.post "#{id}/transfers", options
+    end
+
```

Comment:
no doc on these ?

Reply:
This functions are part of `refund` and `transfer` module

fetch_refund : https://github.com/razorpay/razorpay-ruby/blob/update_refund/documents/refund.md#fetch-a-specific-refund-for-a-payment

fetch_mutiple_refund : https://github.com/razorpay/razorpay-ruby/blob/update_refund/documents/refund.md#fetch-multiple-refunds-for-a-payment

transfers : https://github.com/razorpay/razorpay-ruby/blob/update_transfers/documents/transfers.md#create-transfers-from-payment

Reply:
This functions are part of `refund` and `transfer` module

fetch_refund : https://github.com/razorpay/razorpay-ruby/blob/update_refund/documents/refund.md#fetch-a-specific-refund-for-a-payment

fetch_mutiple_refund : https://github.com/razorpay/razorpay-ruby/blob/update_refund/documents/refund.md#fetch-multiple-refunds-for-a-payment

transfers : https://github.com/razorpay/razorpay-ruby/blob/update_transfers/documents/transfers.md#create-transfers-from-payment